packages/cli/src/commands/dev.js

```diff
@@ -95,11 +100,14 @@ export const handler = async ({
 
   const redwoodConfigPath = getConfigPath()
 
+  const apiDebugPortEnvVar = apiDebugPort
```

Comment:
What if this was a flag instead of an env var?

e.g. 
```js
const getApiDebugFlag = () => {
 // Passed in flag takes precedence
 if (apiDebugPort) {
  return `--debug-port ${apiDebugPort}` 
 }

 const apiDebugPortInToml = getConfig().api.debugPort
 if (apiDebugPortInToml) {
   return `--debug-port ${apiDebugPort}`
 }

 // Dont pass in debug port flag, unless configured
 return ''
}
```

And in the command
```js
command: `yarn cross-env NODE_ENV=development NODE_OPTIONS=--enable-source-maps yarn nodemon --watch "${redwoodConfigPath}" --exec "yarn rw-api-server-watch ${getApiDebugFlag()} | rw-log-formatter"
```

Reply:
Okay, I put in in yargs so there is a --help as well